scripts/chk_shellscripts/chk_shellscripts.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -e
+
+REPO_ROOT="$(dirname "$0")"/../..
+REPO_ROOT=$(realpath "${REPO_ROOT}")
+IGNORE_FILENAME="ignore.txt"
+IGNORE_FILE="${REPO_ROOT}/scripts/chk_shellscripts/${IGNORE_FILENAME}"
+
+IGNORE=()
+while IFS='' read -r line
+do
+  IGNORE+=("$line")
+done < <(
+  cat "${IGNORE_FILE}"
+)
+
+FILES=()
```

Comment:
Same here - why not `for file in find . -name "*.sh"`?

I think we can easily assume that even if the path to the repository might contain spaces, the files in the repository won't.

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2044

Reply:
`This will fail for filenames containing spaces and similar` - we do not have such files.

Reply:
oki.. then this can be simplified